region/maridia/inner-green.json

```diff
@@ -1211,6 +1211,64 @@
                     ]}
                   ],
                   "devNote": "Other weapons take too long to kill the enemy."
+                },
+                {
+                  "name": "Cross Room Jump with Damage Boost",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    "canPlayInSand",
+                    "canDamageBoost",
+                    {"enemyDamage": { "enemy": "Evir", "type": "contact", "hits": 1 }},
+                    {"adjacentRunway": {
+                      "fromNode": 1,
+                      "usedTiles": 7,
+                      "physics": ["air"]
+                    }}
+                  ],
+                  "note": [
+                    "Requires a runway of approximately 7 tiles in the adjacent room.",
+                    "Jump on room entry then damage boost off of the right Evir in order to land on the first pillar."
+                  ],
+                  "devNote": "It is assumed that getting to the second pillar is free relative to getting to the right door."
+                },
+                {
+                  "name": "Cross Room Jump with Air Ball",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    "canPlayInSand",
+                    "canLateralMidAirMorph",
+                    {"adjacentRunway": {
+                      "fromNode": 1,
+                      "usedTiles": 7,
+                      "physics": ["air"]
+                    }}
+                  ],
+                  "note": [
+                    "Requires a runway of approximately 7 tiles in the adjacent room.",
+                    "Jump into an air ball on room entry in order to get to the first pillar."
+                  ],
+                  "devNote": "It is assumed that getting to the second pillar is free relative to getting to the right door."
+                },
+                {
+                  "name": "Cross Room Jump with Bomb Boost and Spring Ball",
+                  "notable": false,
+                  "requires": [
+                    "h_canBombHorizontally",
+                    "canSpringBallBombJump",
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentRunway": {
+                      "fromNode": 1,
+                      "usedTiles": 0.5,
+                      "physics": ["air"]
+                    }}
+                  ],
+                  "note": [
+                    "Only requires a runway of approximately 0.5 tiles in the adjacent room.",
```

Comment:
This doesn't need springball if you bomb a pixel away from the door transition, like how insomnia does it.
Springballjumping through a sandfall should probably require canPlayInSand.

Reply:
I think Bob moved before the explosion hit to get more distance.